pkg/util/util.go

```diff
@@ -96,3 +98,20 @@ func GetCSVfromPod(pod *corev1.Pod, c client.Client, logger logr.Logger) (*csvv1
 
 	return csv, nil
 }
+
+func NewKubeVirtPriorityClass() *schedulingv1.PriorityClass {
+	return &schedulingv1.PriorityClass{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "scheduling.k8s.io/v1",
+			Kind:       "PriorityClass",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "kubevirt-cluster-critical",
```

Comment:
I think that we should also enforce Labels as we are doing for other resources, see:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/pkg/controller/hyperconverged/hyperconverged_controller.go#L760

Reply:
Rethinking about this: we should label the priority class for tracking purposes but we should not try to delete it when the user deletes HCO CR, I'll address this in a separate PR